src/hardware/uni-t-ut8802e/api.c

```diff
@@ -0,0 +1,219 @@
+/*
+ * This file is part of the libsigrok project.
+ *
+ * Copyright (C) 2023 Konrad Tagowski <konrad.tagowski@grinn-global.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <config.h>
+#include "protocol.h"
+
+static const uint32_t scanopts[] = {
+    SR_CONF_CONN,
+};
+
+static const uint32_t drvopts[] = {
+    SR_CONF_MULTIMETER,
+};
+
+static const uint32_t devopts[] = {
+    SR_CONF_CONTINUOUS,
+    SR_CONF_LIMIT_SAMPLES | SR_CONF_GET | SR_CONF_SET,
+    SR_CONF_LIMIT_MSEC | SR_CONF_GET | SR_CONF_SET,
+    SR_CONF_DATA_SOURCE | SR_CONF_GET | SR_CONF_LIST,
+};
+
+
+static const char *channel_names[] = {
+    "Main",
+};
+
+static const char *data_sources[] = {
+    "Live",
+};
```

Comment:
Since this multimeter only has live measurements, i.e. no storage mode, there is no need to implement `SR_CONF_DATA_SOURCE` in this driver. So this can go, and the entry in `devopts[]`, and the `config_get()/config_list()` entries as well.